src/sinks/util/encoding/config.rs

```diff
@@ -67,6 +67,22 @@ where
     }
 }
 
+#[cfg(any(feature = "sinks-new_relic_logs", feature = "sinks-humio"))]
```

Comment:
I'm a little confused why this type only is used by these two sinks and not the others. Should it be?

Reply:
Yes, it's should be here. Only used in New Relic / Humio because we need to convert `EncodingConfig<sink1::Encoding>` to `EncodingConfig<sink2::Encoding>`.

Reply:
Ah! Gotcha, I see.